### DISCUSSION TITLE
need server reboot

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#7251] [General]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

Need help to shutdown and restart server for www.wlcp-db.com.  I can't log in so I need help to be able to shutdown and restart.

**Comment by bombelman** [comment]
option 1: Log into your webhosting panel to reboot, you should have access.
option 2: Contact your web hosting provider to give you (root) access and restart using ssh
option 3: hire a network admin to manage your server :-)

**Reply by ghost** [reply]
My web site is hosted on Linode.  I logged into Linode and shut down server and rebooted.  I am still seeing the error.  Is my only option now to reinstall Azuracast?

**Comment by ghost** [comment]
The website is hosted on cloud flare.  Is that the web hosting panel you are talking about?

**Reply by ghost** [reply]
website is hosted on Linode not Cloudflare.  I bought my domain from Cloudflare and use Cloudflare for performance and security

**Comment by bombelman** [comment]
that is possible yes. only you can know. you should know.
it could also be a DNS issue with cloudflare, as it's giving a error 500.
try restarting there, but not sure it's the issue.
either way, it needs to be looked at.

**Reply by ghost** [reply]
I have determined that my website is hosted on Linode.  I was able to shut down the server and reboot.  Still having the issue.   Is my only option to reinstall Azuracast?

**Comment by ghost** [comment]
Thank you

**Comment by Vaalyn** [comment]
I moved this thread over to discussions as this is not an issue with AzuraCast itself
